tensorboard/uploader/uploader_test.py

```diff
@@ -228,6 +236,138 @@ def scalar_event(tag, value):
         self.assertEqual(4 + 6, mock_client.WriteScalar.call_count)
         self.assertEqual(4 + 6, mock_rate_limiter.tick.call_count)
 
+    # Verify behavior with lots of small chunks
+    @mock.patch.object(uploader_lib, "BLOB_CHUNK_SIZE", 100)
+    def test_start_uploading_graphs(self):
+        mock_client = _create_mock_client()
+        mock_rate_limiter = mock.create_autospec(util.RateLimiter)
+        mock_blob_rate_limiter = mock.create_autospec(util.RateLimiter)
+        uploader = _create_uploader(
+            mock_client,
+            "/logs/foo",
+            rpc_rate_limiter=mock_rate_limiter,
+            blob_rpc_rate_limiter=mock_blob_rate_limiter,
+            allowed_plugins=["scalars", "graphs"],
```

Comment:
Since you’re relying on `graphs` being the actual plugin name, not just
an arbitrary string, could you please use `graphs_metadata.PLUGIN_NAME`?

Reply:
Done.  I left some preexisting instances of `"scalars"` where it doesn't actually matter.  I think `"foobar"` would be clearer that those cases allow arbitrary strings, but this PR isn't the place for that.